share/wake/lib/core/list.wake

```diff
@@ -802,3 +805,76 @@ export def groupBy (cmp: a => a => Boolean) (list: List a): List (List a) =
             (elem, head, rest), tail
         _ = (elem, Nil), acc
     foldr combo Nil sorted
+
+export def fromOption (x: Option a): List a =
+    getOrElse Nil $ omap (_, Nil) x
```

Comment:
I think we should use @ag-eitilt 's name of `optionToList`.

Which is easier for someone to read?
```
export def fromOption: Option a => List a = match _
  Some x = x, Nil
  None = Nil
```

... remember new users are almost certainly going to goto-definition on a function like this.